cinema/serializers.py

```diff
@@ -59,6 +68,9 @@ class MovieSessionListSerializer(MovieSessionSerializer):
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField(
+        read_only=True,
```

Comment:
The `tickets_available` field is defined here but lacks an implementation for calculating available tickets. Ensure this field is properly implemented to reflect the actual number of available tickets.